torch/csrc/autograd/python_variable.cpp

```diff
@@ -144,6 +144,32 @@ static PyObject *THPVariable_pynew(PyTypeObject *type, PyObject *args, PyObject
 }
 
 // Instantiates a subclass of torch.Tensor. Used by nn.Parameter()
+static PyObject* THPVariable_as_subclass(THPVariable* self, PyObject* args, PyObject* kwargs) {
+  HANDLE_TH_ERRORS
+  static PythonArgParser parser({
+    "as_subclass(PyObject* cls)",
+  });
+  ParsedArgs<2> parsed_args{};
+  auto r = parser.parse(args, kwargs, parsed_args);
+  PyObject* cls = r.pyobject(0);
+  if (!PyType_Check(cls)) {
+    throw TypeError("cls must be a type (got %s)", Py_TYPE(cls)->tp_name);
+  }
+  auto data = as_variable_ref(self->cdata);
+  // We set `data`'s `allow_tensor_metadata_change` to true here, because we want to
```

Comment:
This is a completely new function right? Why is there this comment about backwards compatibility?

Reply:
Because it was a copypasta before https://github.com/pytorch/pytorch/pull/34369#issuecomment-599635556.